Apps/W1/EDocumentConnectors/Continia/app/src/Implementation/ExtConnectionSetup.Page.al

```diff
@@ -0,0 +1,162 @@
+                        ApiRequests: Codeunit "Api Requests";
+                        Participations: Page Participations;
+                        ProgressWindow: Dialog;
+                    begin
+                        ProgressWindow.Open(ProcessingWindowMsg);
+                        if Participation.FindSet() then
+                            repeat
+                                ApiRequests.GetParticipation(Participation);
+                                ApiRequests.GetAllParticipationProfiles(Participation);
+                            until Participation.Next() = 0;
+
+                        ProgressWindow.Close();
+                        Participations.Run();
+                    end;
+                }
+            }
+        }
+    }
+
+    actions
+    {
+        area(Promoted)
+        {
+            actionref(RegisterNewActionRef; RegisterNewParticipation) { }
+            group(SubscriptionGroup)
+            {
+                Caption = 'Subscription';
+                actionref(EditCompanyInformationActionRef; EditCompanyInformation) { }
+                actionref(CancelSubscriptionActionRef; CancelSubscription) { }
+            }
+        }
+        area(Processing)
+        {
+            action(RegisterNewParticipation)
+            {
+                Caption = 'Register new participation';
+                Enabled = CanEditParticipation;
+                Image = New;
+                ToolTip = 'Register a new participation in the Continia Delivery Network.';
+
+                trigger OnAction()
+                var
+                    OnboardingWizard: Page "Onboarding Wizard";
+                begin
+                    OnboardingWizard.Run();
+                    CurrPage.Update(false);
+                end;
+            }
+            action(EditCompanyInformation)
+            {
+                Caption = 'Edit Company Information';
+                Enabled = ActionsEnabled and CanEditParticipation;
+                Image = Company;
+                ToolTip = 'Edit the company information for the Continia subscription.';
+
+                trigger OnAction()
+                var
+                    ActivationMgt: Codeunit "Subscription Mgt.";
+                    OnBoardingWizard: Page "Onboarding Wizard";
+                    RunScenario: Enum "Wizard Scenario";
+                begin
+                    if ActivationMgt.HasOtherAppsInSubscription() then
+                        Message(UseContiniaSolutionMgtMsg)
+                    else begin
+                        OnBoardingWizard.SetRunScenario(RunScenario::EditSubscriptionInfo);
+                        OnBoardingWizard.Run();
+                    end;
+                end;
+            }
+            action(CancelSubscription)
+            {
+                Caption = 'Cancel Subscription';
+                Enabled = ActionsEnabled and CanEditParticipation;
+                Image = Cancel;
```

Comment:
Actions that provide a confirm or a request page should show this by using the ellipsis property.